openff/toolkit/topology/topology.py

```diff
@@ -1524,6 +1524,149 @@ def _ensure_unique_atom_names(self, ensure_unique_atom_names: Union[str, bool]):
             elif not molecule.has_unique_atom_names:
                 molecule.generate_unique_atom_names()
 
+    @classmethod
+    @requires_package("openmm")
+    def from_pdb(
+        cls,
+        file_path: Union[str, TextIO],
+        unique_molecules: Optional[Iterable[Molecule]] = None,
+        toolkit_registry=GLOBAL_TOOLKIT_REGISTRY,
+    ):
+        """
+        Loads polymers, waters, monoatomic ions, and user-defined molecules from a PDB file.
+
+        Molecules in the PDB file have the following requirements:
+          * Polymer molecules must use the standard atom names described in the
+            `PDB Chemical Component Dictionary <https://www.wwpdb.org/data/ccd>`_.
```

Comment:
@Yoshanuikabundi - If you have docstring-reviewing energy I'd love a glance at this. Feel free to directly modify+push any changes/polish you like. I'll merge this early on my Monday (if tests pass) so no worries if you don't get to it before then.

Reply:
(But I am NOT asking for a full PR review, this one is big and complex so I'm self-approving it as product owner. But a glance at the docstring would be great if you are in the mood)